src/main/java/org/jenkins/plugins/lockableresources/LockableResourceProperty.java

```diff
@@ -0,0 +1,66 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * Copyright (c) 2013, 6WIND S.A. All rights reserved.                 *
```

Comment:
Is this a new file? And do you work for 6WIND? I am not sure (IANAL) if the copyrights for new stuff that at best tangentially reuses old stuff are required to reference prior art authors. Probably does not hurt to have this here, just wondering what the practical requirements are, gray area for me...

Reply:
Will remove in #299